neo/rawio/tdtrawio.py

```diff
@@ -181,15 +209,24 @@ def _parse_header(self):
                         assert self._sigs_lengths[seg_index][stream_index] == size
 
                     # signal start time, relative to start of segment
-                    t_start = data_index['timestamp'][0]
+                    if len(data_index['timestamp']):
+                        t_start = data_index['timestamp'][0]
+                    else:
+                        # if no signal present use segment t_start as dummy value
+                        t_start = self._seg_t_starts[seg_index]
```

Comment:
Is it initialize

Reply:
Yes, `seg_t_starts` is already initialized in line 128 & 130 and contains either a number or np.nan.

Reply:
If no event or signal or other timepoints are present in the segment, then this will probably lead to an error as a Segment shouldn't have `np.nan` as t_start. However, this was already previously the case (see lines 128 & 130).